File: ProductOrders/Scripts/models/orderDocument.js
var OrderDocument = function (orderRecordId, type, number, date, amount, currency, extendedInformation) {
    var self = this;

    self.orderRecordId = ko.observable(orderRecordId);
    self.type = ko.observable(type);
    self.number = ko.observable(number);
    self.date = ko.observable(date);
    self.amount = ko.observable(amount);
    self.currency = ko.observable(currency);
    self.extendedInformation = ko.observable(extendedInformation);
    self.isExtendedInfoExpanded = ko.observable(false);

    return self;
};

File: ProductOrders/Models/OrderDocument.cs
using System.ComponentModel.DataAnnotations;

namespace ProductOrders.Models
{
    public class OrderDocument
    {
        [Key]
        public int OrderRecordId { get; set; }
        public string Type { get; set; }
        public string Number { get; set; }
        public string Date { get; set; }
        public string Amount { get; set; }
        public string Currency { get; set; }
        public string ExtendedInformation { get; set; }
    }
}